composio/sdk/sdk.py

```diff
@@ -17,12 +17,36 @@
     NotFoundException,
     TimeoutException,
 )
-from composio.sdk.http_client import HttpClient
+from composio.sdk.local_tools import LocalToolHandler
+from composio.sdk.http_client import HttpClient, HttpHandler
 
 from .enums import Action, App, Tag
+from .local_enums import LocalAction, LocalApp
 from .storage import get_base_url
 
 
+class ActionEnum(Enum):
+    def __init__(self, service, action, no_auth):
+        self.service = service
+        self.action = action
+        self.no_auth = no_auth
+
+
+# Collect all members from both enums
+combined_members = {name: member.value for name, member in Action.__members__.items()}
+combined_members.update({name: member.value for name, member in LocalAction.__members__.items()})
+
+# Dynamically create a new enum class with the combined members
+Action = ActionEnum('Action', combined_members)
+
+# Collect all members from both App enums
+combined_app_members = {name: member.value for name, member in App.__members__.items()}
+combined_app_members.update({name: member.value for name, member in LocalApp.__members__.items()})
+
+# Dynamically create a new App enum class with the combined members
+App = Enum('App', combined_app_members)
```

Comment:
This is going to add a lot of runtime overhead, need to find a better way to do this.